libs/application/templates/financial-statements-inao/src/lib/messages.ts

```diff
@@ -0,0 +1,170 @@
+  },
+  infoSection: {
+    id: 'fsn.application:intro.section',
+    defaultMessage: 'Upplýsingar',
+    description: 'Info',
+  },
+  fundInfo: {
+    id: 'fsn.application:fund',
+    defaultMessage: 'Upplýsingar um sjóð og vörsluaðila',
+    description: 'fund info',
+  },
+  about: {
+    id: 'fsn.application:about',
+    defaultMessage: 'Um þig',
+    description: 'about',
+  },
+  email: {
+    id: 'fsn.application:person.email',
+    defaultMessage: 'Netfang',
+    description: 'email',
+  },
+  phoneNumber: {
+    id: 'fsn.application:about.phoneNumber',
+    defaultMessage: 'Símanúmer',
+    description: 'phone number',
+  },
+  keyNumbers: {
+    id: 'fsn.application:keyNumbers',
+    defaultMessage: 'Lykiltölur',
+    description: 'Statement key numbers',
+  },
+  properties: {
+    id: 'fsn.application:properties',
+    defaultMessage: 'Eignir',
+    description: 'List of applicants properties',
+  },
+  propertiesShort: {
+    id: 'fsn.application:properties.short',
+    defaultMessage: 'Skammtímakröfur',
+    description: 'Short term',
+  },
+  propertiesCash: {
+    id: 'fsn.application:properties.cash',
+    defaultMessage: 'Bankainnistæður og sjóðir',
+    description: 'Cash is king',
+  },
+  income: {
+    id: 'fsn.application:income',
+    defaultMessage: 'Tekjur',
+    description: 'Applicants income',
+  },
+  donations: {
+    id: 'fsn.application:income.donations',
+    defaultMessage: 'Framlög lögaðila',
+    description: 'Donations',
+  },
+  personalIncome: {
+    id: 'fsn.application:income.personal',
+    defaultMessage: 'Eigin tekjur',
+    description: 'personal income',
+  },
+  capitalIncome: {
+    id: 'fsn.application:income.capital',
+    defaultMessage: 'Fjármagnstekjur',
+    description: 'capital income',
+  },
+  financialStatement: {
+    id: 'fsn.application:financial.statment',
+    defaultMessage: 'Ársreikningur',
+    description: 'financial statements',
+  },
+  upload: {
+    id: 'fsn.application:upload',
+    defaultMessage: 'Hlaða upp ársreikningi',
+    description: 'Upload financial statements',
+  },
+  uploadIntro: {
+    id: 'fsn.application:upload.intro',
+    defaultMessage: 'Vinsamlegast hlaðið upp ársreikning hér að neðan.',
+    description: 'Upload financial statements intro',
+  },
+  uploadDescription: {
+    id: 'fsn.application:upload.description',
+    defaultMessage: 'Vinsamlegast hlaðið upp ársreikning hér að neðan.',
+    description: 'Upload financial statements intro',
+  },
+  confirm: {
+    id: 'fsn.application:confirmation',
+    defaultMessage: 'Staðfesta',
+    description: 'confirm application',
+  },
+  send: {
+    id: 'fsn.application:send',
+    defaultMessage: 'Senda umsókn',
+    description: 'Send application',
+  },
+  overviewTitle: {
+    id: 'fsn.application:overviewTitle',
+    defaultMessage: 'Yfirlit',
+    description: 'Title for overview section',
+  },
+  overviewSectionTitle: {
+    id: 'fsn.application:overviewTitle',
```

Comment:
This isnt an unique id and is going to clash with the overviewTitle once you run extract strings on your application. Which you cant now as you dont have the application defined in the workspace.json